python/opae.admin/opae/admin/fpga.py

```diff
@@ -218,6 +218,16 @@ def spi_bus(self):
                     return spi
         return self.find_one('spi*/spi_master/spi*/spi*')
 
+    @property
+    def pmci_bus(self):
+        if os.path.basename(self.sysfs_path).startswith('dfl'):
+            patterns = ['dfl*.*/*n6010bmc-secure*']
```

Comment:
Rather than **n6010bmc-secure**, could we just use **-secure**?  This would make things forward compatible.